go.mod

```diff
@@ -47,7 +47,7 @@ require (
 	github.com/nsf/termbox-go v1.1.1
 	github.com/olekukonko/tablewriter v0.0.5
 	github.com/pkg/errors v0.9.1
-	github.com/prologic/bitcask v0.3.9
+	git.mills.io/prologic/bitcask v0.3.9
```

Comment:
v0.3.14?

Reply:
Yes! Please. Sorry I was being lazy and did these PR(s) with a shell script en-mass. Please upgrade to 0.3.14 -- We are releasing v1 this weekend 🎉

Reply:
> Yes! Please. Sorry I was being lazy and did these PR(s) with a shell script en-mass. Please upgrade to 0.3.14 -- We are releasing v1 this weekend 🎉

@prologic ok, cool, thanks for the feedback! Will do. And congrats to v1!